webpack.common.js

```diff
@@ -1,14 +1,58 @@
+const { resolve } = require('path');
 const webpack = require('webpack');
-const path = require('path');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CopyPlugin = require('copy-webpack-plugin');
+
+const pythonPath = resolve(__dirname, 'pydata_sphinx_theme');
+const staticPath = resolve(pythonPath, 'static');
+
+const vendor = resolve(staticPath, 'vendor');
+const vendorVersions = {
+  fontAwesome: require("@fortawesome/fontawesome-free/package.json").version,
+  openSans: require("@openfonts/open-sans_all/package.json").version,
+  lato: require("@openfonts/lato_latin-ext/package.json").version
+};
+const vendorPaths = {
+  fontAwesome: resolve(vendor, 'fontawesome', vendorVersions.fontAwesome),
+  openSans: resolve(vendor, 'open-sans_all', vendorVersions.openSans),
+  lato: resolve(vendor, 'lato_latin-ext', vendorVersions.lato)
+};
+
+// generates cache-busting templates to be used in `layout.html` without knowing versions
```

Comment:
not really that proud of this, but have used similar approaches before to pass That One Bit of information between languages/timescales.